client/app/(authenticated)/bceidbusiness/industry_user/dashboard/operations/page.tsx

```diff
@@ -1,6 +1,11 @@
 // 🚩 flagging that for shared routes between roles, "Page" code is a component for code maintainability
 import OperationsPage from "@/app/components/routes/operations/Page";
+import { OperationsSearchParams } from "@/app/components/routes/operations/types";
 
-export default async function Page() {
-  return <OperationsPage />;
+export default async function Page({
+  searchParams,
+}: {
+  searchParams: OperationsSearchParams;
+}) {
+  return <OperationsPage searchParams={searchParams} />;
```

Comment:
These page files are all refactored because for reasons I don't understand, there is no hook or easy way to get `search params` in server components in NextJS besides this way. Search params are pass to all `page.tsx` props so I am passing them through so we can use them to fetch data in the server components.